app/src/main/java/com/fstyle/androidtrainning/screen/main/MainActivity.java

```diff
@@ -1,24 +1,29 @@
 package com.fstyle.androidtrainning.screen.main;
 
 import android.os.Bundle;
+import android.support.annotation.IntDef;
 import android.support.v4.view.ViewPager;
 import android.support.v7.app.AppCompatActivity;
 import android.support.v7.widget.SearchView;
 import android.support.v7.widget.Toolbar;
 import android.view.Menu;
 import android.view.MenuItem;
 import com.fstyle.androidtrainning.R;
+import com.fstyle.androidtrainning.utils.Constant;
```

Comment:
ERROR: Unused import: com.fstyle.androidtrainning.utils.Constant. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports